tests/settings_tests/tests.py

```diff
@@ -382,6 +382,23 @@ def test_use_deprecated_pytz_deprecation(self):
         ):
             holder.configure(USE_DEPRECATED_PYTZ=True)
 
+    def test_unable_to_import_settings_module(self):
+        with self.assertRaisesMessage(
+            ImproperlyConfigured,
+            "Settings module fake_settings_module could not be imported",
+        ):
+            Settings("fake_settings_module")
+
+    def test_unable_to_import_a_random_module(self):
+        def mock_import_module(_):
+            raise ModuleNotFoundError(
+                "No module named 'fake_module'", name="fake_module"
+            )
+
```

Comment:
This can be handled with the `side_effect`:
```python
        exc = ModuleNotFoundError("No module named 'fake_module'", name="fake_module")
        with mock.patch("importlib.import_module", side_effect=exc):
            with self.assertRaisesMessage(ImportError, "No module named 'fake_module'"):
                Settings("fake_settings_module")
```